Defs/Stats/Stats_Pawns_Combat.xml

```diff
@@ -3,6 +3,63 @@
 
   <!-- ================================== Shooting =======================================-->
 
+  <StatDef>
+    <defName>AimingDelayFactor</defName>
```

Comment:
Would it not be safer/potentially less destructive to other mods to patch these changes into the vanilla stat, rather than overwriting it entirely?